packages/react-menu/src/components/Menu/Menu.types.ts

```diff
@@ -26,6 +26,13 @@ export interface MenuProps extends MenuListProps {
    * Wrapper to style and add events for the popup
    */
   menuPopup?: ShorthandProps<React.HTMLAttributes<HTMLElement>>;
+
+  /**
+   * How the menu should be opened
+   *
+   * @defaultvalue click
+   */
+  on?: ('click' | 'hover' | 'focus' | 'context')[];
```

Comment:
I suggest to check what are possible combinations there, for example in v0 this type is more restricted as not all combinations of events are meaninful:

https://github.com/microsoft/fluentui/blob/4503e2d3f62f785363b5937bfb5ce278c27ab0fe/packages/fluentui/react-northstar/src/components/Popup/Popup.tsx#L53-L56

Reply:
I am actually going to remove this prop